app/src/main/java/com/example/android/sunshine/app/Utility.java

```diff
@@ -259,6 +260,36 @@ public static int getArtResourceForWeatherCondition(int weatherId) {
         return -1;
     }
 
+    /**
+     * Helper method to provide the string according to the weather
+     * condition id returned by the OpenWeatherMap call.
+     *
+     * @param context   Android context
+     * @param weatherId from OpenWeatherMap API response
+     * @return string for the weather condition. null if no relation is found.
+     */
+    public static String getStringForWeatherCondition(Context context, int weatherId) {
+        // Based on weather code data found at:
+        // http://bugs.openweathermap.org/projects/api/wiki/Weather_Condition_Codes
+        int stringId;
+        if (weatherId >= 200 && weatherId <= 232) {
+            stringId = R.string.condition_2xx;
+        } else if (weatherId >= 300 && weatherId <= 321) {
+            stringId = R.string.condition_3xx;
+        } else {
+            stringId = context.getResources().getIdentifier(
+                    String.format(Locale.US, "condition_%d", weatherId),
+                    "string",
+                    BuildConfig.APPLICATION_ID);
+
+            if (stringId == 0) {
```

Comment:
sounds like an if-else `if string == 0 do this otherwise do ->` https://github.com/eduardb/Sunshine-Version-2/pull/1/files#diff-7537c0d5330829deab3c57d11a53aa5dR280